timemachine/cpp/src/local_md_potentials.cu

```diff
@@ -13,13 +13,17 @@
 
 namespace timemachine {
 
+namespace {
+
 // Struct representing the CUB < operation
 struct LessThan {
     int compare;
     CUB_RUNTIME_FUNCTION __device__ __forceinline__ explicit LessThan(int compare) : compare(compare) {}
     CUB_RUNTIME_FUNCTION __device__ __forceinline__ bool operator()(const int &a) const { return (a < compare); }
 };
 
+} // namespace
```

Comment:
Unsure/question: is it possible use `static` here to stick with the convention of static over unnamed namespaces?

Reply:
No, can't define static structs/classes covered in https://stackoverflow.com/questions/4422507/how-are-unnamed-namespaces-superior-to-the-static-keyword